src/type-generator.ts

```diff
@@ -89,7 +114,7 @@ export class TypeGenerator {
       let typedColumnName = columnName;
       let columnType = `${info.type.type}`;
 
-      const isColumnOptional = info.nullable || info.hasDefaultValue;
+      const isColumnOptional = info.nullable;
```

Comment:
This shouldn't break any test as insertion depends on the specified columns. It was removed because the type for fields with default values was always optional thus resulting in folks having to do type assertion.